docs/4. Product Features/WhatsApp Forms.md

```
@@ -0,0 +1,230 @@
+  height="315"
+  src="https://drive.google.com/file/d/1Uedfm3JDLnkhSHjVcUrLZolUixo4RUkX/preview"
+  frameborder="0"
+  allow="autoplay"
+  allowfullscreen>
+</iframe>
+
+### How WhatsApp Form Results Come to BigQuery
+1. Ensure your Glific account is connected to BigQuery instance
+2. There are two new tables `whatsapp_forms` and `whatsapp_forms_responses`
+3. `whatsapp_forms` stores the forms and their statuses
+4. ‘whatsapp_forms_responses`	stores the responses recorded on each user
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 30

(MD010, no-hard-tabs)